x/attribute/wasm/encode.go

```diff
@@ -91,6 +125,34 @@ func (params *DeleteAttributeParams) Encode(contract sdk.AccAddress) ([]sdk.Msg,
 	return []sdk.Msg{msg}, nil
 }
 
+// Encode creates a MsgDeleteDistinctAttribute.
+// The contract must be the owner of the name of the attribute being deleted.
+func (params *DeleteDistinctAttributeParams) Encode(contract sdk.AccAddress) ([]sdk.Msg, error) {
+	if err := types.ValidateAttributeAddress(params.Address); err != nil {
+		return nil, fmt.Errorf("wasm: invalid address: %w", err)
+	}
+	msg := types.NewMsgDeleteDistinctAttributeRequest(params.Address, contract, params.Name, params.Value)
+	return []sdk.Msg{msg}, nil
+}
+
+// Encode creates a MsgDeleteAttribute.
```

Comment:
Minor typo. I believe it should read "Encode creates a MsgUpdateAttributeRequest".

Reply:
fixed